models/gc_picai_baseline/meta.json

```diff
@@ -0,0 +1,138 @@
+    "inputs": [
+      {
+        "label": "Prostate biparametric MRI",
+        "description": "Prostate biparametric MRI exam",
+        "format": "DICOM",
+        "modality": "MR",
+        "bodypartexamined": "Prostate",
+        "slicethickness": "",
+        "non-contrast": false,
+        "contrast": false
+      }
+    ],
+    "outputs": [
+      {
+        "type": "Prediction",
+        "valueType": "Probability",
+        "label": "Prostate cancer probability",
+        "description": "Case-level likelihood of harboring clinically significant prostate cancer, in range [0,1]",
+        "classes": []
+      },
+      {
+        "type": "Prediction",
+        "valueType": "Probability map",
+        "label": "Transverse cancer detection map",
+        "description": "Detection map of clinically significant prostate cancer lesions in 3D, where each voxel represents a floating point in range [0,1]",
+        "classes": []
+      }
+    ],
+    "model": {
+      "architecture": "3d fullres nnUNet",
+      "training": "supervised",
+      "cmpapproach": "3D"
+    },
+    "data": {
+      "training": {
+        "vol_samples": 1200
+      },
+      "evaluation": {
+        "vol_samples": 300
+      },
+      "public": false,
+      "external": false
+    }
+  },
+  "details": {
+    "name": "PI-CAI challenge baseline",
+    "version": "v2.1.1",
+    "devteam": "Diagnostic Image Analysis Group, Radboud University Medical Center, Nijmegen, The Netherlands",
+    "type": "Prediction",
+    "date": {
+      "weights": "2022-06-22",
+      "code": "2022-09-05",
+      "pub": ""
+    },
+    "cite": "",
+    "license": {
+      "code": "Apache 2.0",
+      "weights": "Apache 2.0"
+    },
+    "publications": [],
+    "github": "https://github.com/DIAGNijmegen/picai_nnunet_semi_supervised_gc_algorithm",
+    "zenodo": "",
+    "colab": "",
+    "slicer": false
+  },
+  "info": {
+    "use": {
+      "title": "Intended use",
+      "text": "Prediction of the likelihood of harboring clinically significant prostate cancer (csPCa) in prostate biparametric MRI exams.",
+      "references": [
+        {
+          "label": "PI-CAI baseline algorithm on grand-challenge",
+          "uri": "https://grand-challenge.org/algorithms/pi-cai-baseline-nnu-net-semi-supervised/"
+        }
+      ],
+      "tables": []
+    },
+    "analyses": {
+      "title": "Evaluation",
+      "text": "Patient-level diagnosis performance is evaluated using the Area Under Receiver Operating Characteristic (AUROC) metric. Lesion-level detection performance is evaluated using the Average Precision (AP) metric. Overall score used to rank each AI algorithm is the average of both task-specific metrics: Overall Ranking Score = (AP + AUROC) / 2",
+      "references": [
+        {
+          "label": "PI-CAI AI challenge details",
+          "uri": "https://pi-cai.grand-challenge.org/AI/"
+        }
+      ],
+      "tables": []
```

Comment:
It'd be great to have (the most relevant) evaluation results here.

Reply:
I think the tuning set of 100 cases can be removed here, as it's more used for model tuning rather than testing.

On the PI-CAI testing cohort of 1000 cases, this algorithm achieved an Area Under the Receiver Operating Characteristic curve of 0.865 and an Average Precision of 0.576 (i.e., ranking score of 0.721).

The evaluation run can also be linked directly: https://pi-cai.grand-challenge.org/evaluation/fe187cdb-cb61-4cbb-ab63-2de483a52d60/

Reply:
We have updated this section for the evaluation results on the 1000-cases testing cohort, and added a table with the scores together with the link to the scores.